input/pages/artifact-repository-service.md

```diff
@@ -191,7 +191,18 @@ The _review_ operation supports applying a review to an existing artifact, regar
 
 ##### Approve
 
-The _approve_ operation supports applying an approval to an existing artifact, regardless of status. The operation sets the _date_ and _approvalDate_ elements of the approved artifact, and is otherwise only allowed to add _artifactComment_ elements to the artifact, and to add or update an _endorser_.
+The _approve_ operation supports applying an approval to an existing artifact, regardless of status. The operation sets the _date_ and _approvalDate_ elements of the approved artifact, and is otherwise only allowed to add [_artifactAssessment_](http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-artifactAssessmentArtifact) elements to the artifact, and to add or update an _endorser_.
+
+The following parameters SHOULD be supported for the operation:
+
+* **id**: The server-specific id of the artifact to be approved.
+* **approvalDate**: The date on which the artifact was approved for release. If this parameter is not provided the operation will infer the date to be the current system date on the repository performing the operation.
+* **endorser**: A ContactDetail parameter which should identify (and optionally provide contact information for) the party approving the artifact for release.
```

Comment:
Endorsing is different than approving, we shouldn't need this parameter, since the approvalDate is specifically when the artifact is approved "by the publisher", so the approving party is basically the publisher. I like the addition of the approving party details in the artifact assessment parameters, but I don't think we should have this endorser parameter because 1) we're not endorsing the artifact, and 2) there's nowhere to put it in the artifact (i.e. there is no "approver" element.

Reply:
So, it could be the case that the information about an "endorser" is provided as part of the approval process, that makes sense, but then clarify the documentation here to say that, not that the "endorser" parameter is the "approving party". Sorry for the confusion.

Reply:
Agreed, endorsing should be a separate operation, or it should be a type of "reviewing", we shouldn't conflate it with the "approving" operation